src/shared/components/dropdown_auto_refresh/AutoRefreshDropdown.tsx

```diff
@@ -47,7 +50,7 @@ export default class AutoRefreshDropdown extends Component<Props> {
   }
 
   public render() {
-    if (CLOUD) {
+    if (!this.props.showAutoRefresh) {
```

Comment:
Do we want to keep this exclusively for cloud users?

Reply:
This feature is already on in OSS